sdk/src/api/method/export.rs

```diff
@@ -44,6 +46,112 @@ where
 				name: name.to_owned(),
 				version: version.to_string(),
 			}),
+			db_config: self.db_config,
+			response: self.response,
+			export_type: PhantomData,
+		}
+	}
+
+	/// Whether to export users from the database
+	pub fn with_users(self, users: bool) -> Export<'r, C, R, Model> {
+		let mut db_config = self.db_config.unwrap_or_default();
+		db_config.users = users;
+
+		Export {
+			client: self.client,
+			target: self.target,
+			ml_config: self.ml_config,
+			db_config: Some(db_config),
+			response: self.response,
+			export_type: PhantomData,
+		}
+	}
+
+	/// Whether to export accesses from the database
+	pub fn with_accesses(self, accesses: bool) -> Export<'r, C, R, Model> {
+		let mut db_config = self.db_config.unwrap_or_default();
+		db_config.accesses = accesses;
+
+		Export {
+			client: self.client,
+			target: self.target,
+			ml_config: self.ml_config,
+			db_config: Some(db_config),
+			response: self.response,
+			export_type: PhantomData,
+		}
+	}
+
+	/// Whether to export params from the database
+	pub fn with_params(self, params: bool) -> Export<'r, C, R, Model> {
+		let mut db_config = self.db_config.unwrap_or_default();
+		db_config.params = params;
+
+		Export {
+			client: self.client,
+			target: self.target,
+			ml_config: self.ml_config,
+			db_config: Some(db_config),
+			response: self.response,
+			export_type: PhantomData,
+		}
+	}
+
+	/// Whether to export functions from the database
+	pub fn with_functions(self, functions: bool) -> Export<'r, C, R, Model> {
+		let mut db_config = self.db_config.unwrap_or_default();
+		db_config.functions = functions;
+
+		Export {
+			client: self.client,
+			target: self.target,
+			ml_config: self.ml_config,
+			db_config: Some(db_config),
+			response: self.response,
+			export_type: PhantomData,
+		}
+	}
+
+	/// Whether to export analyzers from the database
+	pub fn with_analyzers(self, analyzers: bool) -> Export<'r, C, R, Model> {
+		let mut db_config = self.db_config.unwrap_or_default();
+		db_config.analyzers = analyzers;
+
+		Export {
+			client: self.client,
+			target: self.target,
+			ml_config: self.ml_config,
+			db_config: Some(db_config),
+			response: self.response,
+			export_type: PhantomData,
+		}
+	}
+
+	/// Whether to export all versions of data from the database
+	pub fn versioned(self, versioned: bool) -> Export<'r, C, R, Model> {
```

Comment:
```suggestion
	pub fn versioned(self, versioned: bool) -> Export<'r, C, R> {
```

Reply:
I think `versioned` makes it sound like the export itself is what's versioned. Have you considered naming this `with_versions`? 🤔